client/js/components/GameView/Card.vue

```diff
@@ -21,6 +21,11 @@
       class="valid-move target-overlay"
       opacity=".8"
     />
+    <transition name="slide-above">
```

Comment:
Do you know if the transition needs to be v-ifed or if if automatically goes away if there's nothing to render? I don't remember and want to make sure we're following convention here. Also seems weird this is using card but the other card below is using an img.

Reply:
An interesting point and I don't know. I'll do some digging

Reply:
Yeah the docs say to use transitions wrapped around elements which themselves have v-if: https://vuejs.org/guide/built-ins/transition.html#the-transition-component